<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editwindow</class>
 <widget class="QDialog" name="editwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>14</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="IdLabel">
      <property name="text">
       <string>Id:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="idtxt">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>478</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>39</y>
     <width>571</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="ObraLabel">
        <property name="text">
         <string>Obra:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="AutorLabel">
        <property name="text">
         <string>Autor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EdLabel">
        <property name="text">
         <string>Edição:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="QntdLabel">
        <property name="text">
         <string>Quantidade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SecaoLabel">
        <property name="text">
         <string>Seção:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PratLabel">
        <property name="text">
         <string>Prateleira:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="ObraLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="AutorLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="EdLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="QntdLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="SecaoLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="PratLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>370</y>
     <width>551</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="GravarBtn">
      <property name="text">
       <string>Gravar</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>378</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="CancelarBtn">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
